src/pages/member/UserHome.js

```diff
@@ -11,58 +10,64 @@ import { makeStyles } from '@material-ui/core/styles'
 import { connect } from 'react-redux'
 
 const useStyles = makeStyles({
-  root: {
-    width: '719px',
-    marginTop: '27px'
+  container: {
+    maxWidth: '1200px',
+    display: 'flex',
+    flexWrap: 'wrap',
+    margin: 'auto',
+    padding: '14px'
   },
-  home: {
+  header: {
     color: COLOR.BIZTECH_GREEN,
-    fontStyle: 'normal',
-    fontWeight: 'bold',
-    paddingLeft: '85px',
-    paddingTop: '50px'
+    width: '100%'
   },
-  page: {
-    height: '100vh'
+  column: {
+    flex: '1'
   },
-  container: {
-    display: 'flex'
+  card: {
+    position: 'relative',
+    margin: '10px 10px 0 0',
+    overflow: 'visible'
   },
-  header: {
-    fontStyle: 'normal',
-    fontWeight: 'bold',
-    paddingLeft: '76px',
-    paddingTop: '45px'
-  },
-  reward: {
-    fontStyle: 'normal',
-    fontWeight: 'normal',
-    fontSize: '22px',
-    paddingLeft: '76px',
-    paddingTop: '15px'
+  flexbox: {
+    display: 'flex',
+    width: '100%'
   },
   house: {
     position: 'absolute',
-    left: '685px',
-    top: '99px'
+    width: '33%',
+    bottom: '0px',
+    right: '10px'
```

Comment:
how does this work? i originally used left and top for absolute position - is this position relative to the CardComponent that the house is in?

Reply:
yeah, the key is https://github.com/ubc-biztech/bt-web/pull/129/files/aa3d01fcec6612a202f4bd2522d4d53a1b40f430#diff-a2f324d958cc036aac267cbe410f2df8R28
the parent has to be set to `relative` position